pkg/cmd/attestation/verify/verify_test.go

```diff
@@ -333,26 +345,29 @@ func TestRunVerify(t *testing.T) {
 	t.Run("with invalid OIDC issuer", func(t *testing.T) {
 		opts := publicGoodOpts
 		opts.OIDCIssuer = "not-a-real-issuer"
+		opts.SigstoreVerifier = &verification.FailSigstoreVerifier{}
 		require.Error(t, runVerify(&opts))
 	})
 
 	t.Run("with SAN enforcement", func(t *testing.T) {
 		opts := Options{
-			ArtifactPath:    artifactPath,
-			BundlePath:      bundlePath,
-			APIClient:       api.NewTestClient(),
-			DigestAlgorithm: "sha512",
-			Logger:          logger,
-			OIDCIssuer:      GitHubOIDCIssuer,
-			Owner:           "sigstore",
-			SAN:             SigstoreSanValue,
+			ArtifactPath:     artifactPath,
+			BundlePath:       bundlePath,
+			APIClient:        api.NewTestClient(),
+			DigestAlgorithm:  "sha512",
+			Logger:           logger,
+			OIDCIssuer:       GitHubOIDCIssuer,
+			Owner:            "sigstore",
+			SAN:              SigstoreSanValue,
+			SigstoreVerifier: verification.NewMockSigstoreVerifier(t),
```

Comment:
Similarly, if we pass in a Mock verifier which always succeeds, what is this test testing?

Reply:
(Again on the impression which may be incorrect that the SAN enforcement happens inside the guts of the LiveSigstore.Verify func which calls out to sigstore-go. I may be misremembering / if it weren't 4pm on a friday I'd do a little bit more digging 😉)

Reply:
Chatted about this test offline, it will be skipped for now and we will add another layer of testing that allows us to more thoroughly test the layer of code that would handle the issues in this test.